alembic/versions/1c87fd8da02e_adds_job_visibility_column.py

```diff
@@ -0,0 +1,142 @@
+            batch_op.create_index('ix_dept_membership_dept_role_dept_role_id', 'dept_membership_dept_role', ['dept_role_id'], unique=False)
+
+        with op.batch_alter_table('dept_membership_request', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_dept_membership_request_attendee_id', 'dept_membership_request', ['attendee_id'], unique=False)
+            batch_op.create_index('ix_dept_membership_request_department_id', 'dept_membership_request', ['department_id'], unique=False)
+
+        with op.batch_alter_table('dept_role', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_dept_role_department_id', 'dept_role', ['department_id'], unique=False)
+
+        with op.batch_alter_table('job_required_role', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_job_required_role_dept_role_id', 'job_required_role', ['dept_role_id'], unique=False)
+            batch_op.create_index('ix_job_required_role_job_id', 'job_required_role', ['job_id'], unique=False)
+
+        with op.batch_alter_table('shift', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_shift_attendee_id', 'shift', ['attendee_id'], unique=False)
+            batch_op.create_index('ix_shift_job_id', 'shift', ['job_id'], unique=False)
+    else:
+        op.add_column('job', sa.Column('visibility', sa.Integer(), server_default='0', nullable=False))
+
+        op.create_index('ix_job_department_id', 'job', ['department_id'], unique=False)
+        op.create_index('ix_dept_membership_attendee_id', 'dept_membership', ['attendee_id'], unique=False)
+        op.create_index('ix_dept_membership_department_id', 'dept_membership', ['department_id'], unique=False)
+        op.create_index('ix_dept_membership_dept_role_dept_membership_id', 'dept_membership_dept_role', ['dept_membership_id'], unique=False)
+        op.create_index('ix_dept_membership_dept_role_dept_role_id', 'dept_membership_dept_role', ['dept_role_id'], unique=False)
+        op.create_index('ix_dept_membership_request_attendee_id', 'dept_membership_request', ['attendee_id'], unique=False)
+        op.create_index('ix_dept_membership_request_department_id', 'dept_membership_request', ['department_id'], unique=False)
+        op.create_index('ix_dept_role_department_id', 'dept_role', ['department_id'], unique=False)
+        op.create_index('ix_job_required_role_dept_role_id', 'job_required_role', ['dept_role_id'], unique=False)
+        op.create_index('ix_job_required_role_job_id', 'job_required_role', ['job_id'], unique=False)
+        op.create_index('ix_shift_attendee_id', 'shift', ['attendee_id'], unique=False)
+        op.create_index('ix_shift_job_id', 'shift', ['job_id'], unique=False)
+
+    # Removes duplicate membership requests caused by a bug that would save a
```

Comment:
This is unrelated to the linked issue, but I added this data cleanup migration because I was already in here messing about with the department code.